File: FR_14_04_Karty/KARTY.py
punkty = {  '2':2,
            '3':3,
            '4':4,
            '5':5,
            '6':6,
            '7':7,
            '8':8,
            '9':9,
            'T':10,
            'J':11,
            'Q':12,
            'K':13,
            'A':14}

j = input()
s = input()
wj=0
ws=0
for x in j:
    wj=wj+punkty[x]

for x in s:
    ws=ws+punkty[x]

if wj>ws:
    print('JASIO')
elif ws<ws:
    print('STASIO')
else:
    print('REMIS')

File: FR_14_04_Karty/KARTY.cpp
/* https://pl.spoj.com/problems/FR_14_04/ */
#include <iostream>
#include <string>
using namespace std;

int punkty(char c){
    string karty="23456789TJQKA";
    return(karty.find(c)+2);
}

int main(){
    string k1, k2;
    int w1, w2;
    w1=0;
    w2=0;
    cin>> k1;
    for (int i;i<k1.length();i++){
    	w1+=punkty(k1[i]);
    }
       
    cin>> k2;
    for (int i;i<k2.length();i++){
        w2+=punkty(k2[i]);
    }
    if (w1>w2){
        cout<<"JASIO"<<endl;
    } else if(w1<w2){
        cout<<"STASIO"<<endl;
    } else {
        cout<<"REMIS"<<endl;
    }
    return 0; 
}